### PR TITLE
readme: add instructions and github workflows badge

### PR DESCRIPTION
**Pull request by ljmf00**
Signed-off-by: Luís Ferreira <contact@lsferreira.net>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
-# autopsy-packager
+# autopsy-packager ![Github Workflows](https://github.com/labcif/autopsy-packager/actions/workflows/packaging.yml/badge.svg?branch=main)
```
Comment (iK4tsu) [inline README.md:1]: I'd prefer if it were before or after the title.

```md
# autopsy-packager
![Github Workflows](https://github.com/labcif/autopsy-packager/actions/workflows/packaging.yml/badge.svg?branch=main)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
+
 Automatic Github Workflows packager for autopsy
+
+## Installation
+
+Before the installation process of `autopsy` package, make sure you have access to the following requirements:
```
Comment (iK4tsu) [inline README.md:7]: Be sure to have the following requirements before starting any installation process of `Autopsy`:

**Review by iK4tsu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ljmf00) [reply README.md]: Sure, done. I added it right after the first heading.

File: README.md (hunk 2 above)
Comment (ljmf00) [reply README.md]: Done

**Comment by ljmf00** [comment]
Additionally, I added a link to workflows list on the Github Workflows badge.

**Comment by iK4tsu** [comment]
LGTM

**Review by iK4tsu — APPROVED** [review]

**Pull request merged**